neon_utils/default_configurations/default_user_conf.yml

```diff
@@ -48,3 +48,8 @@ location:
   country: United States
   tz: America/Los_Angeles
   utc: '-8.0'
+
+speed_mode:
+  mode: quick
+  available_modes: [quick, thoughtful]
```

Comment:
I think `quick` and `thoughtful` can probably just be a comment here, since they should just match keys from `mode_extension_time`